docs/src/user-guide/guides-retention.md

```
@@ -0,0 +1,212 @@
+for and deletes expired data (archives or search results). To understand the high-level algorithm,
+first consider the following definitions:
+
+| Term                | Description                                                                                                                                                                |
+|---------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| $sweep\_interval$   | The interval (in minutes) at which the garbage collector wakes up to check for expired data.                                                                               |
+| $retention\_period$ | The duration (in minutes) for which data (an archive or search result) is retained before it is considered expired.                                                        |
+| $current\_time$     | The time at which the garbage collector is performing a check.                                                                                                             |
+| $data\_timestamp$   | The end of the time range for the data being evaluated for expiration (e.g., for an archive, this is the timestamp of the most recent log event contained in the archive). |
+
+When the garbage collector wakes up, it will scan for and delete any data that fits the expiry
+criteria shown in [Figure 1](#figure-1):
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid

(MD051, link-fragments)